openapi.yaml

```diff
@@ -611,13 +622,15 @@ definitions:
             type: object
             properties:
               name:
-                $ref: '#/definitions/DeveloperResource/properties/name'
+                $ref: '#/definitions/DeveloperResource/properties/attributes/properties/name'
               website:
-                $ref: '#/definitions/DeveloperResource/properties/website'
+                $ref: '#/definitions/DeveloperResource/properties/attributes/properties/website'
   ReviewResource:
     properties:
       id:
         type: string
+      type:
+        $ref: '#/definitions/ReviewType'
       gameId:
         $ref: '#/definitions/GameResource/properties/id'
       reviewer:
```

Comment:
I think these extra properties should be in `ReviewResource.properties.attributes`

Reply:
Fixed this for both GameResource and ReviewResource